app/lib/features/chat_ng/widgets/events/replied_to_event.dart

```diff
@@ -60,10 +60,14 @@ class RepliedToEvent extends StatelessWidget {
       children: [
         ActerAvatar(options: AvatarOptions.DM(replyProfile, size: 12)),
         const SizedBox(width: 5),
-        Text(
-          replyProfile.displayName ?? '',
-          style: Theme.of(context).textTheme.bodySmall?.copyWith(
-            color: Theme.of(context).colorScheme.tertiary,
+        Expanded(
+          child: Text(
+            replyProfile.displayName ?? '',
+            style: Theme.of(context).textTheme.bodySmall?.copyWith(
+              color: Theme.of(context).colorScheme.tertiary,
+            ),
```

Comment:
so, we don't calculate the same color for the display name here than we do in the original message? Whatever we agree to do, we probably should though